dags.yaml

```diff
@@ -361,10 +361,10 @@ bqetl_search_dashboard:
     depends_on_past: false
     email:
       - telemetry-alerts@mozilla.com
-      - ssuh@mozilla.com
+      - bewu@mozilla.com
```

Comment:
@BenWu Do you have context on this search dashboard DAG such that you'd be an appropriate owner? I can assign myself if we don't know who else it would make sense for.

Reply:
I have some context.  I can take this